### PR TITLE
Create is-innersource-right.md

### PR DESCRIPTION
**Pull request by rrrutledge**
View the [rendered markdown](https://github.com/rrrutledge/innersource-program/blob/rrrutledge-patch-3/articles/is-innersource-right.md).

### CODE REVIEW

File: articles/is-innersource-right.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+-   Inviting users to make the contributions that they need and following up with them to ensure that they do them.
+-   Maintaining an CONTRIBUTING.md document that contains everything an engineer needs to know to contribute to the project.
+-   Giving up-front guidance and direction as to how to implement a given contribution.
+-   Being available during regular hours for any ad-hoc questions that contributors have.
+-   Timely review of incoming pull requests.
+-   Ongoing maintenance of submitted code (after the [warranty window](https://patterns.innersourcecommons.org/p/30-day-warranty)).
+
+## Is it Company Specific?
+
+InnerSource projects make sense when the project is specific to the company or gives the company a strategic business advantage. Other collaborative projects should be run as open source so as to increase the contribution pool and impact.
```
Comment (spier) [inline articles/is-innersource-right.md:29]:
```suggestion
InnerSource projects make sense when the project is specific to the company or gives the company a strategic business advantage. Other collaborative projects should be run as open source to increase the contribution pool and impact.
```

simpler language

File: articles/is-innersource-right.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+## Summary
+
+If your project is company-specific *and* contributions will come **and** you will support those contributions, then InnerSource is right for your project.
```
Comment (spier) [inline articles/is-innersource-right.md:33]: Maybe the 3 questions in the same order as they appear in the document above?

To do that, one could move the "Is it Company Specific?" to the top.
Would make sense to me as that should be a simple test/question that needs to be answered with "No" first, before the readers proceed to the next questions.

Comment (spier) [reply articles/is-innersource-right.md:33]:
```suggestion
If your project is company-specific **and** contributions will come **and** you will support those contributions, then InnerSource is right for your project.
```

Did you want to use the same formatting for **and**?

File: articles/is-innersource-right.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
+# Is InnerSource Right for My Project?
+
+InnerSource is the application of open source principles to company-internal software development. Done right, it unblocks progress and eases adoption of shared services and modules. This article contains guidance and questions to ask yourself when considering adoption of an InnerSource approach to running your project.
+
+## Will Contributions Come?
+
+An InnerSource approach only makes sense if contributions are expected from the project's users. You can expect contributions to come if you see or anticipate noticeable amounts of energy directed at your project space by its users. Some examples:
```
Comment (spier) [inline articles/is-innersource-right.md:7]: What does "project space" mean? Why not just "project"?

File: articles/is-innersource-right.md [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+-   Feature requests that take nearly as much time to explain as they would just to implement.
+-   Multiple roadmap dependencies on your project.
+
+## Will I Support Contributions?
+
+Even with willing contributors, the code doesn't just flow in. You will need to encourage and support contributions via activities such as:
+
+-   Understanding users' scenarios and suggesting what contributions in your project could help them to meet those scenarios.
+-   Inviting users to make the contributions that they need and following up with them to ensure that they do them.
+-   Maintaining an CONTRIBUTING.md document that contains everything an engineer needs to know to contribute to the project.
```
Comment (spier) [inline articles/is-innersource-right.md:21]: The is an opportunity link to even more patterns here, if that is desirable.
e.g. https://patterns.innersourcecommons.org/p/base-documentation#contributing.md

**Review by spier — APPROVED** [review]
I really like the document already in its current form.

I left some inline comments for areas that could be improved.

File: articles/is-innersource-right.md (hunk 3 above)
Comment (rrrutledge) [reply articles/is-innersource-right.md:7]: If users are home-rolling workarounds that are in your project space (but not necessarily your exact project) then that is also a good reason to run your project as InnerSource.

**Comment by rrrutledge** [comment]
Thank your for this review, @spier ❗   I moved this content to the [InnerSource Learning Path](https://github.com/InnerSourceCommons/InnerSourceLearningPath/pull/481) and incorporated your suggestions!

**Pull request closed without merging**